fe/fe-core/src/test/java/com/starrocks/http/rest/v2/TablePartitionActionTest.java

```diff
@@ -477,13 +478,12 @@ private static LakeTable newLakeTable(Long tableId, String tableName, int partit
 
     private static RestBaseResultV2<PagedResult<PartitionView>> parseResponseBody(String body) {
         try {
-            System.out.println("resp: " + body);
-            return GsonUtils.GSON.fromJson(
+            return GSON.fromJson(
                     body,
                     new TypeToken<RestBaseResultV2<PagedResult<PartitionView>>>() {
                     }.getType());
         } catch (Exception e) {
-            fail("invalid resp body: " + body);
+            fail(e.getMessage() + ", resp: " + body);
```

Comment:
this file is optional, since the change has nothing to do with your PR.

Reply:
Yes, I found the error message in `TableSchemaActionTest` isn't very intuitive, so I modified it together. Need to rollback?

Reply:
not necessary.